base/reset.css

```diff
@@ -0,0 +1,130 @@
+/* Base */
+
+*,
+*::before,
+*::after {
+  border: 0;
+  box-sizing: border-box;
+  font-family: 'Montserrat', sans-serif;
+  font-size: var(--size-12);
+  margin: 0 0 0 0;
+  scroll-behavior: smooth;
+}
+
+img,
+video {
+  block-size: auto;
+  max-inline-size: 100%;
+}
+
+textarea {
+  background-color: var(--primary-100);
+  border-radius: 8px;
+  height: 8rem;
+  width: 30rem;
+}
+
+html .xsm {
```

Comment:
these classes would be more fitted under the typography component. Just a suggestion.

Reply:
In typography I have added more specific classes reset file contains all the generic classes and resets.